configs/JHEF435PRO/config.h

```diff
@@ -0,0 +1,102 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           JHEF435PRO
+#define MANUFACTURER_ID      JHEF
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_GYRO_CLKIN
+
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_DPS310
+
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+#define USE_MAX7456
+
+#define BEEPER_PIN          PC15
+#define MOTOR1_PIN          PA0  // TMR2_CH1 / TMR5_CH1 / I2C2_SCL / UART4_TX
+#define MOTOR2_PIN          PA1  // TMR2_CH2 / TMR5_CH2 / I2C2_SDA / SPI4_MOSI / UART4_RX
+#define MOTOR3_PIN          PA2  // TMR2_CH3 / TMR5_CH3 / TMR9_CH1 / USART2_TX / SDIO2_CK
+#define MOTOR4_PIN          PA3  // TMR2_CH4 / TMR5_CH4 / TMR9_CH2 / USART2_RX 
+#define LED_STRIP_PIN       PB6  // TMR4_CH1 / I2C1_SCL / SPI4_CS / USART1_TX / UART5_TX
+#define UART1_TX_PIN        PA9  // TMR1_CH2 / SPI2_SCK / USART1_TX / I2C1_SCL
+#define UART2_TX_PIN        NONE 
+#define UART3_TX_PIN        PB10 // TMR2_CH3 / I2C2_SCL / SPI2_SCK / USART3_TX
+#define UART5_TX_PIN        PB9  // TMR2_CH2 / TMR4_CH4 / TMR11_CH1 / I2C1_SDA / SPI2_CS / SPI4_MOSI / I2C2_SDA / UART5_TX
+#define UART1_RX_PIN        PA10 // TMR1_CH3 / SPI2_MOSI / USART1_RX / I2C1_SDA 
+#define UART2_RX_PIN        NONE 
+#define UART3_RX_PIN        PB11 // TMR2_CH4 / TMR5_CH4 / I2C2_SDA / USART3_RX
+#define UART5_RX_PIN        PB8  // TMR2_CH1 / TMR4_CH3 / TMR10_CH1 / I2C1_SCL / SPI4_MISO / UART5_RX
+#define LED0_PIN            PA8  // TMR1_CH1 / I2C3_SCL / USART2_TX 
+#define I2C2_SCL_PIN        PH2  // TMR5_CH1 / I2C2_SCL / UART4_RX
+#define I2C2_SDA_PIN        PH3  // TMR5_CH2 / I2C2_SDA / UART4_TX
+#define SPI1_SCK_PIN        PA5  // TMR2_CH1 / TMR8_CH1C / SPI1_SCK / USART6_RX
+#define SPI2_SCK_PIN        PB13 // TMR1_CH1C / SPI2_SCK / SPI4_SCK / I2C3_SCL
+#define SPI3_SCK_PIN        PB3  // TMR2_CH2 / I2C2_SDA / SPI1_SCK / SPI3_SCK / USART1_RX / UART7_RX
+#define SPI1_SDI_PIN        PA6  // TMR3_CH1 / SPI1_MISO / TMR13_CH1
+#define SPI2_SDI_PIN        PB14 // TMR1_CH2C / TMR8_CH2C / I2C3_SDA / SPI2_MISO
+#define SPI3_SDI_PIN        PB4  // TMR3_CH1 / I2C3_SDA / SPI1_MISO / SPI3_MISO / UART7_TX 
+#define SPI1_SDO_PIN        PA7  // TMR1_CH1C / TMR3_CH2 / TMR8_CH1C / SPI1_MOSI / TMR14_CH1
+#define SPI2_SDO_PIN        PB15 // TMR1_CH3C / TMR8_CH3C / I2C3_SCL / SPI2_MOSI
+#define SPI3_SDO_PIN        PB5  // TMR3_CH2 / SPI1_MOSI / SPI3_MOSI / UART5_RX
+#define ADC_VBAT_PIN        PB0  // TMR1_CH2C / TMR3_CH3 / TMR8_CH2C / USART2_RX / SPI3_MOSI / ADC
+#define ADC_CURR_PIN        PB1  // TMR1_CH3C / TMR3_CH4 / TMR8_CH3C / SPI2_SCK / ADC
+#define PINIO1_PIN          PC13 
```

Comment:
What is the PINIO for? Do we need to set the config and mode?

Reply:
Added